src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.Managed.cs

```diff
@@ -181,6 +171,25 @@ private void PrepareTouchScroll(object sender, ManipulationStartingRoutedEventAr
 			}
 		}
 
+		private void TouchScrollStarted(object sender, ManipulationStartedRoutedEventArgs e)
+		{
+			if (e.Container != this)
+			{
+				// This gesture is coming from a nested element, we just ignore it!
+				return;
+			}
+
+#if HAS_UNO_WINUI
+			if (e.PointerDeviceType == global::Microsoft.UI.Input.PointerDeviceType.Touch)
+#else
+			if (e.PointerDeviceType == global::Windows.Devices.Input.PointerDeviceType.Touch)
+#endif
+			{
+				Debug.Assert(PointerRoutedEventArgs.LastPointerEvent.Pointer.UniqueId == e.Pointers[0]);
+				this.CapturePointer(PointerRoutedEventArgs.LastPointerEvent.Pointer);
```

Comment:
I think I need to release capture in CompleteTouchScroll. Will validate if that produces proper capture lost.